include/shvel-param.h

```diff
@@ -102,7 +102,8 @@ typedef enum
   ERROR_LOW_VOLTAGE = 0x0001,
   ERROR_HALL_SEQ = 0x0002,
   ERROR_HALL_ENC = 0x0004,
-  ERROR_WATCHDOG = 0x0008
+  ERROR_WATCHDOG = 0x0008,
+  ERROR_HEARTBEAT = 0x0010
```

Comment:
https://github.com/T-frog/tf-2md3-firmware/blob/ac0d851e6811b4f8c678ed9d6458f1ac1d80ac7c/tfrog-motordriver/controlVelocity.h#L37-L38
please make it `0x0040`